app/src/main/res/values/strings.xml

```diff
@@ -11,7 +11,7 @@
     <string name="select_app">Select an app</string>
     <string name="select_patches">Select patches</string>
 
-    <string name="unsupported_architecture_warning">Patching on ARMv7 devices is not yet supported and will most likely fail.</string>
+    <string name="unsupported_architecture_warning">Patching on this device architecture is unsupported and will most likely fail.</string>
```

Comment:
If this PR solves patching on ARMv7 then what is this used for? Does it still check if the device arch isnt in the aapt `WORKING_ABIS`?

Reply:
Yes. ARMv7 is added to the array https://github.com/ReVanced/revanced-manager/pull/2084/files#diff-5becc4ed7b2026d3a471cc120219ed7ccfdfdd19a89fbd948f0ff9ef34f65e42R7

Reply:
That's not what I asked, I asked whether or not the ABI check on patch checks if the device ABI is in the array or not, or if it's hardcoded to check for ARMv7